### PR TITLE
Replace mime parser

### PR DESCRIPTION
**Pull request by margori**
To extract urls, php package was used to parser email.

I replaced it with a package that don't requires  that package.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+version: "3.7"
```
Comment (Jamesking56) [inline docker-compose.yml:1]: I don't think this file belongs in this library

File: src/Module/MailCatcher.php [hunk 2]
```diff
@@ -290,22 +291,24 @@ public function grabFromLastEmailTo($address, $regex)
...
         preg_match_all('#\bhttp?://[^,\s()<>]+(?:\([\w\d]+\)|([^,[:punct:]\s]|/))#', $text, $text_matches);
 
-        $html = $parser->getMessageBody('html');
+        $html = $message->getHtmlContent();
         preg_match_all('#\bhttp?://[^,\s()<>]+(?:\([\w\d]+\)|([^,[:punct:]\s]|/))#', $html, $html_matches);
 
-        return array_merge($text_matches[0], $html_matches[0]);
+        $results = [];
+        foreach ($text_matches[0] as $rawResult) {
+            $results[] =quoted_printable_encode($rawResult);
```
Comment (Jamesking56) [inline src/Module/MailCatcher.php:306]:
```suggestion
            $results[] = quoted_printable_encode($rawResult);
```

File: src/Module/MailCatcher.php [hunk 3]
```diff
@@ -290,22 +291,24 @@ public function grabFromLastEmailTo($address, $regex)
...
+        }
+        foreach ($html_matches[0] as $rawResult) {
+            $results[] =quoted_printable_encode($rawResult);
```
Comment (Jamesking56) [inline src/Module/MailCatcher.php:309]:
```suggestion
            $results[] = quoted_printable_encode($rawResult);
```

File: docker-compose.yml (hunk 1 above)
Comment (margori) [reply docker-compose.yml]: Removed

File: src/Module/MailCatcher.php (hunk 2 above)
Comment (margori) [reply src/Module/MailCatcher.php]: Fixed

File: src/Module/MailCatcher.php (hunk 3 above)
Comment (margori) [reply src/Module/MailCatcher.php]: Fixed

**Comment by Jamesking56** [comment]
Thanks @margori :+1:

**Pull request merged**